addons/account/views/audit_trail_report_views.xml

```diff
@@ -55,5 +59,20 @@
         <field name="search_view_id" ref="view_message_tree_audit_log_search"/>
     </record>
 
+    <record id="res_config_settings_view_form_inherit_account_audit_trail" model="ir.ui.view">
+        <field name="name">res.config.settings.form.inherit.account.audit.trail</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="account.res_config_settings_view_form"/>
```

Comment:
It's quite complex because of the dependencies. If we do that we'd have to reorder where to put settings file in the __manifest__ data, and when I tried that there's another dependency error...